charts/rs-server-cadip/templates/configmap.yaml

```diff
@@ -35,9 +35,9 @@ data:
 
         search:
           type: ODataV4Search
-          api_endpoint: '{{ $v.endpoint.url.file }}'
+          api_endpoint: '{{ $v.endpoint.url.odata }}/Files'
           need_auth: true
-          results_entry: responses | $
+          results_entry: $.value
```

Comment:
@vprivat-ads @Padeanu in rs-server we have 4 template session files: 

1. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config.template.yaml
2. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config.template_session.yaml
3. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config_token_module.template.yaml
4. https://github.com/RS-PYTHON/rs-server/blob/fix/rspy491-cadip-search-fix-rebased-3-matthieu/services/cadip/config/cadip_ws_config_token_module.template_session.yaml

I don't know well the difference between the 4, but Vincent in 2 of them you left `results_entry: responses | $` and in the two others you changed to: `results_entry: $.value` 

But here in rs-helm you used `results_entry: $.value` in the 4 occurrences, so which one is right ?